onnxruntime/python/tools/quantization/onnx_quantizer.py

```diff
@@ -590,7 +596,7 @@ def quantize_bias_static(self, bias_name, input_name, weight_name):
 
         return quantized_bias_name
 
-    def quantize_inputs(self, node, indices, initializer_use_weight_qType=True):
+    def quantize_inputs(self, node, indices, initializer_use_weight_qType=True, reduce_range=False, per_channel=False, axis=-1):
```

Comment:
>per_channel [](http://example.com/codeflow?start=100&length=11)

Could you rename it to op_level_per_channel?

Reply:
Changed.